app/containers/settingPage/index.js

```diff
@@ -13,6 +13,9 @@ class SettingPage extends Component {
   renderAboutSection () {
     const licenseList = []
     Object.keys(LicenseInfo).forEach(item => {
+      if (item.startsWith(appInfo.name)) {
```

Comment:
See if you want to check for appInfo.name or appInfo.productName or check for both name and version